server.rb

```diff
@@ -28,13 +28,26 @@
                   }
                 end
 
+def mail_to(to)
+  return ENV['MAIL_TO'] unless ENV['MAIL_TO'].nil?
+  return "#{login}@#{hostname}" if ENV['NUMBER_TO_EMAIL_MAP'].nil?
+
+  # convert env var into hash
+  h = ENV['NUMBER_TO_EMAIL_MAP'].gsub(/[{}:]/,'')
+                                .gsub(/'/,'')
+                                .split(', ')
+                                .map{|h| h1,h2 = h.split('=>'); {h1.strip! => h2.strip!}}
+                                .reduce(:merge)
```

Comment:
Hmm, that looks ugly... how about requiring a proper JSON hash there and just doing `JSON.parse`?